README.md

```diff
@@ -250,6 +251,11 @@ var Foo = require('foo'); // "declarationKeyword": "var"
 const Foo = require('foo'); // "declarationKeyword": "const"
 ```
 
+### `globals`
```

Comment:
The tricky thing about this bit of configuration is that different parts of an application might have different globals. For instance, in tests, you might expect that globals like `describe` and `it` are available, but you don't want the same assumption to be made about non-test code. The nice thing about eslint is there is file-specific and directory-specific configuration to deal with this issue. Do you have a plan for how to deal with this for import-js?